### DISCUSSION TITLE
where is Song View Sound Menu - to disable midi follow?

### DISCUSSION THREAD

**Discussion by saynotohugs** [SynthstromAudible/DelugeFirmware#1410] [Q&A] [answered]
I'm having some problems with midi feedback sending out random CC messages - can anyone tell me where I can disable midi follow/midi feedback?

**Comment by sapphire-arches** [comment, marked as answer]
MIDI follow is in the global settings menu. So from any view, `SHIFT+SELECT ENCODER` will bring up the menu, then scroll to `MIDI`, enter the MIDI submenu, then `MIDI-FOLLOW > CHANNEL` to get to the channel menus, and enter in to each of `A`, `B`, and `C` and turn the select knob until it reads `UNASSIGNED` to unassign the channel.  Alternatively, you can go `SETTINGS > MIDI > MIDI-FOLLOW > FEEDBACK > CHANNEL` and set that to `UNASSIGNED` so no MIDI feedback events are emitted but MIDI follow still works.

**Reply by saynotohugs** [reply]
thanks so much sapphire-arches! 

i would never have guessed that a feature on/off button were SO hidden, wow :) 

the only thing for a, b and c i can find is 
"use any device
device unassigned" and from there i (have to) select
"mpe upper zone" since I don't have any devices that support MPE

anyway I think the midi feedback possibility is really interesting, it's just too complicated for me to find out how it works. And since I get all kinds of mystery CC's coming out of the machine it's great that there's a way to turn it off.

**Reply by sapphire-arches** [reply]
huh, there should be an unassigned option, you shouldn't need to use MPE that way. if it's not working that way we probably have a bug somewhere

**Reply by seangoodvibes** [reply]
> thanks so much sapphire-arches! 
> 
> 
> 
> i would never have guessed that a feature on/off button were SO hidden, wow :) 
> 
> 
> 
> the only thing for a, b and c i can find is 
> 
> "use any device
> 
> device unassigned" and from there i (have to) select
> 
> "mpe upper zone" since I don't have any devices that support MPE
> 
> 
> 
> anyway I think the midi feedback possibility is really interesting, it's just too complicated for me to find out how it works. And since I get all kinds of mystery CC's coming out of the machine it's great that there's a way to turn it off. 

To turn off midi-follow all together, midi follow channels A/B/C should be set to unassigned. You can also use shift + learn in those menu's to unassigned the channel's quickly.

If you want to use midi follow but not midi feedback, then go to the midi feedback > channel menu and set the channel to NONE. It only lets you selected between NONE, A, B, C.

If you want to use midi feedback, make sure it's sending midi feedback on a channel that does not disrupt your midi setup. Eg only send midi feedback on a channel that goes back to your controller and not other devices (eg external synths).

**Reply by saynotohugs** [reply]
Hey seangoodvibes,

Thanks a lot for helping me out :) 

For midi follow, unfortunately there's no way to select "unassigned" on my deluge once it's already set to something. (1.0.1 but the nightly I had on it yesterday was the same thing). But your tip with shift+learn did the trick! 

Same thing for the midi feedback - I cannot set it to unassigned once it it set to something. But also here your tip worked, thanks so much :) 

I have to look into the midi feedback and try it out at some point, so the note about having it only send out to a channel that goes back to my controller and not other devices (eg external synths) makes sense, thanks. 

I also want to say that I really truly appreciate that you brought up the midi feedback idea and made it happen!

**Reply by seangoodvibes** [reply]
> Hey seangoodvibes,
> 
> 
> 
> Thanks a lot for helping me out :) 
> 
> 
> 
> For midi follow, unfortunately there's no way to select "unassigned" on my deluge once it's already set to something. (1.0.1 but the nightly I had on it yesterday was the same thing). But your tip with shift+learn did the trick! 
> 
> 
> 
> Same thing for the midi feedback - I cannot set it to unassigned once it it set to something. But also here your tip worked, thanks so much :) 
> 
> 
> 
> I have to look into the midi feedback and try it out at some point, so the note about having it only send out to a channel that goes back to my controller and not other devices (eg external synths) makes sense, thanks. 
> 
> 
> 
> I also want to say that I really truly appreciate that you brought up the midi feedback idea and made it happen! 

Are you sure you can't scroll to unassigned with the select encoder for the midi follow A/B/C channels
And scroll to none for the midi feedback channel?

While I'm happy that shift +
learn worked. Scrolling to unassigned and none should also work.

Note: you can only select scroll to channel unassigned. You cannot scroll the device to unassigned. To fully unassign device + channel you need to use shift + learn.

Also just another note: midi follow did not exist on 1.0.1 so im not sure what "can't scroll to unassigned" issue you're referring to.

Thanks

**Reply by seangoodvibes** [reply]
Out of curiosity, to better understand the issues you're facing, what is your current midi setup? E.g. what devices are you using together with the deluge and how are they connected to the deluge?

Thanks!
